src/GongSolutions.WPF.DragDrop/DefaultDropHandler.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.ComponentModel;
 using System.Linq;
+using System.Reflection;
```

Comment:
We don't need to add this `using` as both uses of `BindingFlags` below has the namespace included.
```suggestion
```